judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/UrielApplication.java

```diff
@@ -67,5 +69,19 @@ public void run(UrielApplicationConfiguration config, Environment env) {
         env.jersey().register(component.contestSupervisorResource());
         env.jersey().register(component.contestRatingResource());
         env.jersey().register(component.versionResource());
+
+        scheduleJobWithFixedDelay(component.contestScoreboardUpdater(), env, 0, 3, TimeUnit.SECONDS);
+    }
+
+    private void scheduleJobWithFixedDelay(
```

Comment:
Let's extract this into a method in a utility class (e.g. `JudgelsApplications`)